### ISSUE TITLE
Overenthusistic

### ISSUE THREAD

**Issue by krainboltgreene** [atom/language-mustache#6]
When I have this package enabled and try to use handlebar packages this package gets used first for `file.hbs` signatures.

**Comment by taylon** [comment]
Hey @kevinsawicki 

I think this issue needs some attention. I was about to start improving the Handlebars support (which is currently really bad) so I came to check the issues and pull request and saw this, now I'm not sure if this package is really supposed to support Handlebars.

If it is I'll start the pull requests for it, otherwise I think that silvenon's pull request makes a lot of sense, and then we should have another package for Handlebars, and the description of this one should not mention that it supports Handlebars.

**Comment by taylon** [comment]
Just a heads up, I went ahead and greatly improved the [atom-handlebars](https://atom.io/packages/atom-handlebars) package ([pull request](https://github.com/Sneagan/atom-handlebars/pull/27)). It now supports handlebars for real.

I'm not sure, but I don't think that supporting both Handlebars and Mustache in only one package would work well. Since we now have great support for handlebars in the atom-handlebars package, if we keep the hbs and handlebars filetypes in this package, users of atom-handlebars would have to either disable language-mustache (which is what I did for now) or set the grammar manually, which I think is a bad experience.
